### DISCUSSION TITLE
CyberChef CSP issues

### DISCUSSION THREAD

**Discussion by ctr49** [Security-Onion-Solutions/securityonion#4873] [Unsupported Versions] [answered]
On a fresh SO 2.3.60 installation (with "IP" address access mode) I cannot load CyberChef. The static portion of the page loads, but the content security prevents further resources to be loaded.
I don't know if the CSP is set by the reverse proxy or CyberChef itself, but on modern browsers (enforcing CSP) this results in the page being unable to load.
<img width="456" alt="image" src="https://user-images.githubusercontent.com/1297771/126499980-2420c7c9-1738-4077-9a29-4f2df7a23fc9.png">

**Comment by 8inaryMata1eao** [comment]
Same issue here

**Comment by dougburks** [comment, marked as answer]
We've identified the issue and will have a fix soon!

**Reply by dougburks** [reply]
This should be fixed in Security Onion 2.3.61:
https://blog.securityonion.net/2021/07/security-onion-2361-now-available.html

**Reply by lncc63** [reply]
Thanks.  

And by the way, the problem does not exist using the MS Edge browser.  It was with Firefox that I encountered the problem.
